### PR TITLE
chat sanitization and shortcuts

### LINKED ISSUE #2300: chat sanitization and shortcuts

### PR DESCRIPTION
**Pull request by QuietlyWhisper**
Resolves #2300

### CODE REVIEW

File: Resources/Locale/en-US/_CM14/cm-chatsan.ftl [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+medic = tall healer
+doc = tall healer
+doctor = tall healer
+squad leader = tall leader
+sl = tall leader
+commanding officer = tall queen
+co = tall queen
+xo = young tall queen
+executive officer = young tall queen
+so = young tall queen
```
Comment (Hmeister-real) [inline Resources/Locale/en-US/_CM14/cm-chatsan.ftl:61]: wouldn't this pose issues with normal speech? like saying "so what?"

File: Resources/Locale/en-US/_CM14/cm-chatsan.ftl [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+engineer = tall builder
+comtech = tall builder
+combat technitian = tall builder
+corpsman = tall healer
+medic = tall healer
+doc = tall healer
+doctor = tall healer
+squad leader = tall leader
+sl = tall leader
+commanding officer = tall queen
```
Comment (Hmeister-real) [inline Resources/Locale/en-US/_CM14/cm-chatsan.ftl:57]: i would add comm officer too

File: Resources/Locale/en-US/_CM14/cm-chatsan.ftl [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+co = tall queen
+xo = young tall queen
+executive officer = young tall queen
```
Comment (Hmeister-real) [inline Resources/Locale/en-US/_CM14/cm-chatsan.ftl:60]: i would add exec officer

File: Resources/Locale/en-US/_CM14/cm-chatsan.ftl (hunk 1 above)
Comment (QuietlyWhisper) [reply Resources/Locale/en-US/_CM14/cm-chatsan.ftl:61]: Thanks

File: Resources/Locale/en-US/_CM14/cm-chatsan.ftl (hunk 2 above)
Comment (QuietlyWhisper) [reply Resources/Locale/en-US/_CM14/cm-chatsan.ftl:57]: Thanks

File: Resources/Locale/en-US/_CM14/cm-chatsan.ftl (hunk 3 above)
Comment (QuietlyWhisper) [reply Resources/Locale/en-US/_CM14/cm-chatsan.ftl:60]: Thanks

**Pull request merged**